gmerge

```diff
@@ -70,13 +70,13 @@ class GMerger(object):
     try:
       result = urllib2.urlopen(self.devkit_url + '/build',
                                data=self.GeneratePackageRequest(package_name))
-      print result.read()
+      sys.stdout.write(result.read())
```

Comment:
Oops, I forgot that `print` implies a trailing `\n` where `.write` doesn't.  I can fix it in a later pass; I don't think this should hold up a release.